coreclient/src/conversations/messages/persistence.rs

```diff
@@ -20,16 +21,16 @@ use crate::{ContentMessage, ConversationId, ConversationMessage, Message, store:
 use super::{ErrorMessage, EventMessage};
 
 const UNKNOWN_MESSAGE_VERSION: u16 = 0;
-const CURRENT_MESSAGE_VERSION: u16 = 1;
+const CURRENT_MESSAGE_VERSION: u16 = 2;
 
 #[derive(Serialize, Deserialize)]
-struct VersionedMessage {
+pub(crate) struct VersionedMessage {
```

Comment:
We should avoid serializing this data and storing it in db, because content is serialized another time. Instead it is better to introduce a column `content_version` in the `conversation_messages` table and use it to decide how to deserialize the content. Another approach might be just to use an enum here.